packages/reporter/src/studio/StudioTestHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import { getFilenameParts } from '../lib/util'
+import Button from '@cypress-design/react-button'
+import { IconArrowLeft } from '@cypress-design/react-icon'
+import { OpenFileInIDEButton } from '../header/OpenFileInIDEButton'
+
+// this is set up to be stubbed in tests
+export const TestActions = {
+  handleBackButton: () => {
```

Comment:
Is this just canceling studio? If so, we should look into just emitting the `studio:cancel` event.